src/e2e/java/teammates/e2e/pageobjects/InstructorCourseEnrollPage.java

```diff
@@ -113,6 +113,9 @@ public void verifyResultsPanelContains(StudentAttributes[] expectedNewStudents,
         // number of tables depends on what results are present
         int numTables = 0;
         List<WebElement> tables = resultsPanel.findElements(By.tagName("table"));
+        if (expectedErrorStudents != null) {
```

Comment:
The order of the tables was changed and now the error table is now the first table. Error students need to be checked first because of the way the tables are retrieved by `numTables`.